packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -347,7 +347,6 @@ class _AppBarState extends State<AppBar> {
 
     if (parentRoute?.isCurrent ?? true) {
       final Brightness brightness = widget.brightness ?? themeData.primaryColorBrightness;
-      // TODO(jonahwilliams): remove once we have platform themes.
       switch (defaultTargetPlatform) {
         case TargetPlatform.iOS:
```

Comment:
ditto

Reply:
Implemented, but this broke a test and I'm unsure of how to fix